code/game/objects/items/plushes.dm

```diff
@@ -546,6 +546,15 @@
 	gender = FEMALE
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
 
+/obj/item/toy/plush/georgefloyd
+	name = "George Floyd"
+	desc = "PLEAS I CAMT BREATHE."
+	icon_state = "george_floyd"
+	item_state = "george_floyd"
+	attack_verb = list("took breath away from")
+	gender = MALE
+	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
+
```

Comment:
This is cursed, at least try to actually pay respects not be a cuck about it homie

Reply:
it says F to pay respects